### DISCUSSION TITLE
Updating /Future installations

### DISCUSSION THREAD

**Discussion by techiegirl2day** [OpenSageTV/Community#14] [Q&A] [answered]
With the shutdown of old forum, what will happen for new installs with plugins?  Should I move on to alternative services?  I prefer SageTV over other options. I was planning to reinstall my Win 10 server after end of recording season to fix a corrupted WIn 10 install. Will I still be able to do that?  What are my options? 

Galaxysurfer

**Comment by jusjoken** [comment, marked as answer]
The change to the forum will not affect the ability to use plugins.  All usable plugins are hosted on GitHub and have been for some time as well are the Windows Installers.  Looking up install processes and moving between searching the old forums and the new forums may be more awkward (especially for new users) but the forum change should not affect the software we use.

Ken

**Reply by techiegirl2day** [reply]
Ok just needed confirmation of changes Thanks. I had read in notice of things shutting down.  Just learning the new lay of the land..  So you are saying that all will be carrying on as we knew it for most part, except from a different location for forum part.

**Reply by valnar1** [reply]
Yeppers!
